tests/integration/targets/lightsail_static_ip/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+static_ip_name: "{{ resource_prefix }}_static_ip"
+region: "us-west-2"
```

Comment:
```suggestion
```
Our CI uses us-east-1, and tests should generally run in whatever region is in the configuration supplied by the user running them (usually via cloud-config-aws.ini).